crates/c_api/src/types/table.rs

```diff
@@ -79,18 +80,30 @@ pub extern "C" fn wasm_tabletype_new(
 
 /// Returns a shared reference to the element type of the [`wasm_tabletype_t`].
 #[no_mangle]
+#[cfg_attr(
+    feature = "prefix-symbols",
+    export_name = "wasmi_wasm_tabletype_element"
+)]
```

Comment:
This should use the `#[cfg_attr(feature = "prefix-symbols", wasmi_c_api_macros::prefix_symbol)]` attribute. Probably an oversight due to text replacement without multi-lines.

Also many more occurrences in the same file.

Reply:
Another day, another mess with `:substitute`!